reference/3.0/Microsoft.PowerShell.Core/About/about_Parameters.md

```diff
@@ -102,24 +101,19 @@ parameter attributes.
 
 This setting indicates whether the parameter is mandatory, that is, whether
 all commands that use this cmdlet must include this parameter. When the value
-is **"True"** and the parameter is missing from the command, Windows PowerShell
+is **"True"** and the parameter is missing from the command, PowerShell
 prompts you for a value for the parameter.
 
 #### Parameter Position
 
-This setting indicates whether you can supply a parameter's value without
-preceding it with the parameter name. If set to **"0"** or **"named"**, a
-parameter name is required. This type of parameter is referred to as a
-named parameter. A named parameter can be listed in any position after the
+If the `Position` setting is set to a positive integer, the parameter name is
+not required. This type of parameter is referred to as a positional parameter,
+and the number indicates the position in which the parameter must appear in
+relation to other positional parameters. A named parameter can be listed in any
+position after the cmdlet name. If you include the parameter name for a
+positional parameter, the parameter can be listed in any position after the
 cmdlet name.
 
-If the `Parameter position` setting is set to an integer other than **"0"**, the
-parameter name is not required. This type of parameter is referred to as a
-positional parameter, and the number indicates the position in which the
-parameter must appear in relation to other positional parameters. If you
-include the parameter name for a positional parameter, the parameter can be
-listed in any position after the cmdlet name.
-
 For example, the `Get-ChildItem` cmdlet has Path and Exclude parameters. The
 `Parameter position` setting for `Path` is **"1"**, which means that it is a
```

Comment:
If changing them all from `parameter position` to `position`, missed this one.

Reply:
Fixed. And I found another thing that needed changing.